packages/block-library/src/details/edit.js

```diff
@@ -56,6 +56,16 @@ function DetailsEdit( { attributes, setAttributes, clientId } ) {
 							} )
 						}
 					/>
+					<TextControl
+						label="Name"
```

Comment:
Do we have any precedence for not translating a label when it refers to the name of a HTML attribute?

Reply:
Good catch, this should be translatable.